app/app/Http/Controllers/LocationReportController.php

```diff
@@ -35,7 +35,8 @@ public function show(string $location_id, $rating_system = null)
 			$rating_system = 'universal';
 		}
 		$location = Location::find($location_id);
-		$question_categories = QuestionCategory::with('questions')->get();
+		//changes order by
```

Comment:
Remove the "changes order by" comment because that doesn't seem to clear up any confusion for a developer who looks at it.

Reply:
Done.!